osquery/tables/system/windows/chassis_info.cpp

```diff
@@ -0,0 +1,69 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/windows/wmi.h"
+
+namespace osquery {
+namespace tables {
+
+QueryData genChassisInfo(QueryContext& context) {
+  QueryData results;
+
+  WmiRequest wmiSystemReq("select * from Win32_SystemEnclosure");
+  const auto& wmiResults = wmiSystemReq.results();
+  if (wmiSystemReq.getStatus().ok()) {
+    if (!wmiResults.empty()) {
+      for (const auto& data : wmiResults) {
+        Row r;
+        bool boolean = false;
```

Comment:
Rename this :p Also auto

Reply:
I struggled to find a good name for this since it's so generic. Any ideas here? What if I just said "condition"? Or maybe "isPresent" ? That sounds very c# though

Reply:
Perhaps an endrun, these seem to be more commonly represented as INTs

Reply:
You know, revisiting this after a long hiatus made me realize "boolean" in this case relates to items in wmiResults, not a variable that is tracking if I want to bail out.

I'm renaming this to isPresent, because it will represent if a certain feature exists on the machine (wmiResults will have true/false for this field). If there's any thought on a better variable name, let me know.

Consequently, I'm going to add a bail condition to the for loop.